<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="2" total="7" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="178784" started-at="2016-03-02T21:10:20Z" finished-at="2016-03-02T21:13:18Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="178784" started-at="2016-03-02T21:10:20Z" finished-at="2016-03-02T21:13:18Z">
      <class name="com.norwex.pws.PwsShop">
        <test-method status="PASS" signature="Authenticate()[pri:0, instance:com.norwex.pws.PwsShop@73846619]" name="Authenticate" is-config="true" duration-ms="11337" started-at="2016-03-02T15:10:20Z" finished-at="2016-03-02T15:10:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Authenticate -->
        <test-method status="FAIL" signature="SetAccount()[pri:1, instance:com.norwex.pws.PwsShop@73846619]" name="SetAccount" duration-ms="40645" started-at="2016-03-02T15:10:31Z" finished-at="2016-03-02T15:11:12Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.norwex.pws.PwsCreate.SignUp(PwsCreate.java:38)
	at com.norwex.pws.PwsShop.SetAccount(PwsShop.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetAccount -->
        <test-method status="PASS" signature="Shop()[pri:2, instance:com.norwex.pws.PwsShop@73846619]" name="Shop" duration-ms="165" started-at="2016-03-02T15:11:12Z" finished-at="2016-03-02T15:11:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Shop -->
        <test-method status="PASS" signature="OrderItems()[pri:6, instance:com.norwex.pws.PwsShop@73846619]" name="OrderItems" duration-ms="17273" started-at="2016-03-02T15:11:12Z" finished-at="2016-03-02T15:11:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderItems -->
        <test-method status="PASS" signature="UpdateCart()[pri:8, instance:com.norwex.pws.PwsShop@73846619]" name="UpdateCart" duration-ms="16345" started-at="2016-03-02T15:11:29Z" depends-on-methods="com.norwex.pws.PwsShop.OrderItems" finished-at="2016-03-02T15:11:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UpdateCart -->
        <test-method status="PASS" signature="CheckOut()[pri:10, instance:com.norwex.pws.PwsShop@73846619]" name="CheckOut" duration-ms="47590" started-at="2016-03-02T15:11:45Z" depends-on-methods="com.norwex.pws.PwsShop.OrderItems" finished-at="2016-03-02T15:12:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckOut -->
        <test-method status="FAIL" signature="PayForOrders()[pri:12, instance:com.norwex.pws.PwsShop@73846619]" name="PayForOrders" duration-ms="45400" started-at="2016-03-02T15:12:33Z" depends-on-methods="com.norwex.pws.PwsShop.CheckOut" finished-at="2016-03-02T15:13:18Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.norwex.pws.PwsShop.PayForOrders(PwsShop.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PayForOrders -->
        <test-method status="SKIP" signature="DisplaySummary()[pri:14, instance:com.norwex.pws.PwsShop@73846619]" name="DisplaySummary" duration-ms="0" started-at="2016-03-02T15:13:18Z" depends-on-methods="com.norwex.pws.PwsShop.PayForOrders" finished-at="2016-03-02T15:13:18Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method PwsShop.DisplaySummary()[pri:14, instance:com.norwex.pws.PwsShop@73846619] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method PwsShop.DisplaySummary()[pri:14, instance:com.norwex.pws.PwsShop@73846619] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DisplaySummary -->
      </class> <!-- com.norwex.pws.PwsShop -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
